components/dashboard/src/projects/Prebuild.tsx

```diff
@@ -167,7 +167,7 @@ export default function () {
                         />
                     </div>
                     <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
-                        {prebuildInstance && <PrebuildInstanceStatus prebuildInstance={prebuildInstance} />}
+                        {prebuildInstance && <PrebuildStatus prebuild={prebuild!} />}
```

Comment:
Shouldn't this be:

```suggestion
                        {prebuild && <PrebuildStatus prebuild={prebuild} />}
```

Reply:
Indeed. I wonder, why we use the `WorkspaceInstance` at all in `PrebuildLogs`. The call site can use `onPrebuildUpdate` instead, so we even can get rid of a subsequent API call 👍🏻